lib/stale.js

```diff
@@ -80,7 +80,68 @@ module.exports = class Stale {
 
     const query = queryParts.join(' ')
     const days = this.getConfigValue(type, 'days') || this.getConfigValue(type, 'daysUntilStale')
-    return this.search(type, days, query)
+    const results = await this.search(type, days, query)
+
+    const pinned = await this.getPinnedNumbers(type)
+    const nonPinned = results.data.filter(issue => pinned.includes(issue.number))
+    return {
+      ...results,
+      data: {
+        ...results.data,
+        total_count: nonPinned.length,
+        items: nonPinned,
+      }
+    }
```

Comment:
I opted to preserve any untouched portions of the return payload, rather than just return `{data:{items: nonPinned}}`, despite the fact they aren't used.

It felt weird to change the return type, rather than just the contents.